human_aware_rl/static/human_data/README.md

```diff
@@ -0,0 +1,97 @@
+# Human Experiment Data
+
+This subdirectory is the home of all human experiment data for the Overcooked game. All data was collected through Mturk and is fully anonymized. While the data collection code is proprietary, it relies heavily on the open source [Overcooked-demo](https://github.com/HumanCompatibleAI/overcooked-demo) project. 
+
+Data was collected on behalf of the Center for Human-Compatible AI (CHAI) at UC Berkeley. Do not distribute in any manner without the express consent of CHAI or its affiliates. If you have questions regarding data rights, privacy, or distrubion please contact either Nathan Miller at nathan_miller23@berkeley.edu or Micah Carroll at mdc@berkeley.edu. 
+
+# Overview 
+
+* [Data directory structure](#data-directory-structure)
+* [Schema](#Schema)
+* [Processing utils](#processing-utils)
+
+# Data Directory Structure
+
+This directory is subdivided into three subdirectories as follows 
+
+`human_data/`
+* `raw/`
+  * Contains all unprocessed, unfiltered data in CSV form
+  * Data is divided into 2019 experiments, collected for [this](https://arxiv.org/abs/1910.05789) paper, and 2020 experiments, collected on more complex layouts with updated dynamics
+* `cleaned/`
+  * Contains processed, filted data as pickled pandas DataFrames
+  * Data is again divided into 2019 and 2020 experiments
```

Comment:
```suggestion
  * Data is again divided into 2019 (changed to be forward compatible with new dynamics) and 2020 experiments
```

Reply:
The 2019 data (as it's stored in the repo currently) doesn't actually have the dynamics updates. The schema for the CSV, as well as the OvercookedState serialization, was forward ported, but the necessary 'interact's for soup cooking are not included

Reply:
The `process_human_data.py` script includes out-of-the-box functionality for adding those additional interact actions.

My thought process was that, since we include the script to add the interacts (but don't include, or rather, don't have, a script for removing them) we should include the raw data that is missing the additional interacts. This way, users can run the script to add them should they so desire, or work with the original trajectories directly should they so choose